test/test-lib/supertest.ts

```diff
@@ -1,23 +1,26 @@
 import * as express from 'express';
 import * as _ from 'lodash';
-import * as supertest from 'supertest';
+import * as $supertest from 'supertest';
```

Comment:
Is this a standard pattern? (adding a `$` in same named modules)

Reply:
Honestly it's not something we do often enough to really have a pattern for, that said we do use `$` for a bunch of other places where we want a private/public version with the same name, eg for memoized functions